staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2723,8 +2723,25 @@ func describeService(service *corev1.Service, endpoints *corev1.Endpoints, event
 		w.Write(LEVEL_0, "Type:\t%s\n", service.Spec.Type)
 		w.Write(LEVEL_0, "IP:\t%s\n", service.Spec.ClusterIP)
 
-		if service.Spec.IPFamily != nil {
-			w.Write(LEVEL_0, "IPFamily:\t%s\n", *(service.Spec.IPFamily))
+		if service.Spec.IPFamilyPolicy != nil {
```

Comment:
this should go before ClusterIP, so we get:

Policy
IPFamilies
ClusterIP
ClusterIPs